daemon/lxd.go

```diff
@@ -166,3 +169,75 @@ func (l *LXD) DeleteContainer(hostname string) (bool, error) {
 
 	return true, nil
 }
+
+func (l *LXD) CreateContainerBootstrapFile(container pfmodel.Container, fullPath string) error {
```

Comment:
Instead of passing fullPath here, should we just generate the random path in this function instead?